packages/angular/src/abstract-control.ts

```diff
@@ -86,6 +86,14 @@ export abstract class JsonFormsAbstractControl<
     this.triggerValidation();
   }
 
+  shouldShowUnfocusedDescription(): boolean {
+    const config = this.jsonFormsService.getState().jsonforms.config;
```

Comment:
The `config` is just an easy way to set default options for ui schema elements. If the same option exists on a ui schema element it has a higher precedence then the one from the config.